Tests/ForwardCompatTestTrait.php

```diff
@@ -0,0 +1,82 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Bundle\MonologBundle\Tests;
+
+use PHPUnit\Framework\TestCase;
+
+// Auto-adapt to PHPUnit 8 that added a `void` return-type to the setUp/tearDown methods
```

Comment:
I suggest using the phpunti-bridge instead, which deals with this

Reply:
I couldn't find the trait in the bridge, but I will look again.

Reply:
I have restored the custom Trait as tests with --prefer-lowest will fetch a version of the bridge, which does not have the trait.